docs/contributing.rst

```diff
@@ -177,6 +192,10 @@ merge into a single tool; see `ongoing discussion
 .. _`pypa-dev mailing list`: https://groups.google.com/forum/#!forum/pypa-dev
 .. _`virtual environment`: https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/
 .. _`tox`: https://tox.readthedocs.io/en/latest/
+.. _`isort`: https://timothycrosley.github.io/isort/
+.. _`black`: https://black.readthedocs.io/en/stable/
+.. _`flake8`: https://flake8.pycqa.org/en/latest/
+.. _`mypy`: https://mypy.readthedocs.io/en/stable/
```

Comment:
How do we decide whether to point to the `stable` docs, or the `latest` docs?

Reply:
Ah, good question. I just used the URL that was linked from the project. It's probably more flexible to use the hostname, and let their configuration handle the redirect.

Done in https://github.com/pypa/twine/pull/631/commits/a4db2fc2f099711383f3a13175356f780d7d2625

Reply:
Yes, linking to the homepage and let it handle the redirect is a good idea.